src/utils/xhr-utils.js

```diff
@@ -205,7 +205,11 @@ export function getViewerInterceptResponse(win, ampdocSingle, input, init) {
   }
   const viewer = Services.viewerForDoc(ampdocSingle);
   const whenFirstVisible = viewer.whenFirstVisible();
-  if (isProxyOrigin(input) || !viewer.hasCapability('xhrInterceptor')) {
+  if (
+    isProxyOrigin(input) ||
```

Comment:
Doesn't this case cover ww.js?

Reply:
right, the issue is with local testing and I think in that case we can just use mode.localdev, mode.test as a signal to bypass intercept rather than as a property in the fetch init.

Reply:
Let's make sure that `bypassInterceptor` can only used in localDev by renaming it to `bypassInterceptForDev` and changing the condition below to `(init.bypassInterceptForDev && getMode().localDev)`.

This reduces the risk that someone sets it by accident and we let an XHR leak un-intercepted in prod.